src/explore-education-statistics-admin/src/hooks/useCKEditorConfig.ts

```diff
@@ -29,13 +37,14 @@ const useCKEditorConfig = ({
 }: {
   allowComments?: boolean;
   allowedHeadings?: string[];
-  editorInstance?: MutableRefObject<Editor | undefined>;
+  editorInstance?: MutableRefObject<EditorType | undefined>;
   glossaryItems?: {
     title: string;
     slug: string;
     body: string;
   }[];
-  toolbarConfig?: string[];
+  includePlugins?: ReadonlySet<PluginName>;
+  toolbarConfig?: readonly ToolbarOption[];
```

Comment:
Similar to `FormEditor`, we could make these types more flexible by allowing the non-readonly types as well